tests/test_vxlan_vtep.rb

```diff
@@ -110,24 +110,21 @@ def test_description
     assert_equal(vtep.description, desc, "description is not #{desc}")
   end
 
-  def test_mac_distribution
+  def test_host_reachability
     vtep = VxlanVtep.new('nve1')
 
-    val = :flood
-    vtep.mac_distribution = val
-    assert_equal(vtep.mac_distribution, val.to_s,
-                 "mac_distribution is not #{val}")
+    vtep.host_reachability = 'flood'
+    assert_equal(vtep.host_reachability, 'flood',
+                 "host_reachability is not 'flood'")
```

Comment:
fyi: I tend to not include an error message for `assert_equal` calls since on failure they'll display `test_host_reachability` `Expected: 'flood'` `Found: 'foobar'` so the message usually doesn't add much value. But it's fine as is.